### PR TITLE
4207 refactor chipgroups

### PR DESCRIPTION
**Pull request by Liliya-web**

### CODE REVIEW

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/ChipGroup.java [hunk 1]
```diff
@@ -80,9 +80,4 @@ public String color() {
     public ChipGroupAssert is() {
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/ChipGroup.java:80]: add IsColor fro ChipGroup and delete color() methods

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/ChipGroupsTests.java [hunk 2]
```diff
@@ -82,13 +95,22 @@ public void slideChipGroupTest() {
...
                 EXPECTED_CHIP_TEXTS.get(4));
+        multipleChipGroup.show();
         multipleChipGroup.select(valuesToTest);
         multipleChipGroup.is().selected(valuesToTest);
         multipleChipGroup.deselect(valuesToTest);
         multipleChipGroup.is().deselected(valuesToTest);
     }
+
+    @Test
+    public void mandatoryChipGroupTest() {
```
Comment (omega790) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/ChipGroupsTests.java:108]: To show how work with select()/deselect() methods will be enough to check with one value and with several values.
This test is excessive.

**Review by omega790 — CHANGES_REQUESTED** [review]

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java [hunk 3]
```diff
@@ -63,11 +64,4 @@ public ChipGroupAssert column() {
         jdiAssert(element().isColumn() ? "column" : "not column", Matchers.is("column"));
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java:64]: Please refactor all methods:
- delete part with ternary operator
- add error message for assert

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java [hunk 4]
```diff
@@ -17,7 +18,7 @@
  * Assertions for {@link ChipGroup}
  */
 public class ChipGroupAssert extends UIAssert<ChipGroupAssert, ChipGroup>
-        implements ThemeAssert<ChipGroupAssert, ChipGroup> {
+        implements ThemeAssert<ChipGroupAssert, ChipGroup>, ColorAssert<ChipGroupAssert, ChipGroup> {
 
     @JDIAction("Assert that {name} is disabled")
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java:23]: Wrong description

**Review by omega790 — CHANGES_REQUESTED** [review]

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java (hunk 3 above)
Comment (Liliya-web) [reply jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java]: done except methods which call ChipAssert

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java [hunk 5]
```diff
@@ -17,14 +19,16 @@
...
  */
 public class ChipGroupAssert extends UIAssert<ChipGroupAssert, ChipGroup>
-        implements ThemeAssert<ChipGroupAssert, ChipGroup> {
+        implements ThemeAssert<ChipGroupAssert, ChipGroup>, ColorAssert<ChipGroupAssert, ChipGroup> {
 
-    @JDIAction("Assert that {name} is disabled")
+    @JDIAction("Assert that '{name}' has values '{0}'")
     public ChipGroupAssert text(List<String> values) {
-        jdiAssert(element().groupElements().stream()
+        Set<String> actualValues = element().groupElements().stream()
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java:26]: Please move part with getting actual chips text to ChipGroup

**Review by omega790 — CHANGES_REQUESTED** [review]

**Review by omega790 — APPROVED** [review]

**Pull request merged**